source/Playnite/Localization/LocSource.xaml

```diff
@@ -38,13 +38,14 @@ Indeterminate - No changes (when editing multiple games)</sys:String>
     <sys:String x:Key="LOCCrashWindowTitle">Oh no, something bad happened…</sys:String>
     <sys:String x:Key="LOCCrashDescription" xml:space="preserve">Unrecoverable error occurred.
         
-If you want to help us fix this issue, please create diagnostics package and use 'Report Crash' button to let us know what happened. Thank you.</sys:String>
+If you want to help us fix this issue, please briefly describe actions taken before the crash, create a diagnostics package, and use the 'Report Crash' button to let us know what happened. Thank you.</sys:String>

```

Comment:
I think this whole string needs to be changed completely. If the user provides description of crash then it seems unnecessary to ask them to open github issue as well, where they will be asked to provide description again.

Reply:
I have modified the message. Please take a look and let me know what you think.